content/api/vision/_index.md

```diff
@@ -25,7 +25,5 @@ A marker's position can be represented using both the [Cartesian](coordinates/#c
 {{% /notice %}}
 
 ## Marker PDFs
-You can download PDF files of all markers:
```

Comment:
Please can you cherry-pick https://github.com/sourcebots/docs/pull/147/commits/0f5f0a4bf45a7d351a8e3347c43ba4e35fc34f02 (removing the marker links) to the proper branch?